data/datasets_labels/cityscapes19.labels

```diff
@@ -0,0 +1,20 @@
+road
```

Comment:
I have two suggestions for this directory:

1. I think we should adjust the terminology. In machine learning, labels are output variables corresponding to the data instances, and that's not what we have here. (For example, the _labels_ for ImageNet 2012 are in `val.txt`.) These are human-readable representations of classes represented by those labels, so I think this directory should be called `dataset_classes`.

2. The names of the files should be made uniform. I think just `<dataset>.txt` (or `<dataset>.json`, although IMO demos that take classes in JSON format should be updated to use the text format instead) makes the most sense.

Reply:
not sure we need deep folder structure (although I'm ok with this).

@IRDonch support uniform naming, but I think replacing json with txt files probably better to do in separate PR after we finish with this one.

Reply:
done

Reply:
> support uniform naming, but I think replacing json with txt files probably better to do in separate PR after we finish with this one.

Yes, I definitely agree.